app/models/reservation.rb

```diff
@@ -18,7 +18,8 @@ class Reservation < ActiveRecord::Base
   validate :matched_item_and_model
   validate :check_status
   validate :status_final_state
-  validate :not_in_past, :available, :check_banned, on: :create
+  validate :not_in_past, :available, :check_banned,
+           :check_consecutive, on: :create
```

Comment:
Note that we probably don't want to check those validations all the time since it will mean that we won't be able to edit a reservation manually or approve requests or update status, so we'll have to think about how best to handle this.